plugins/docker/compose_cli.ml

```diff
@@ -27,12 +30,14 @@ end
 
 module Outcome = Current.Unit
 
-let cmd args { Key.docker_context; name; detach=_ } =
+let cmd args { Key.docker_context; name; detach=_; up_args=_ } =
   Cmd.docker ~docker_context (["compose"; "-f"; "/dev/stdin"; "-p"; name ] @ args)
 
 let cmd_pull = cmd ["pull"]
 
-let cmd_update ({ Key.detach; _ } as key) = cmd ("up" :: if detach then ["-d"] else []) key
+let cmd_update ({ Key.detach; up_args; _ } as key) =
+    let args = if detach then ["-d"] else [] @ up_args in
```

Comment:
The `@ up_args` is binding inside the `else` branch, is this intended? It looks like it could be intended to bind outside the `if` statement.

If it is correct then `[] @` can be removed, LGTM.

Reply:
Also the indentation is inconsistent for this function compared to the rest of the module.